README.md

```diff
@@ -566,6 +566,22 @@ estas restricciones).
 
 ## 6. Accidentes y esencia
 
+Brooks considera como dificultades "principales" las inherentes a la naturaleza
+del software y el resto las clasifica como "accidentales".
+
+Normalmente usaremos los términos en el mismo sentido, pero preferimos comenzar
+considerando a la complejidad del problema en sí misma incluso antes de haber
+introducido al software en el contexto. Es por ello que definimos los siguientes
+dos tipos de complejidad:
+
+- **Complejidad Esencial** es inherente y la esencial del *problema* (como se
+  percibe por los usuarios).
+
+- La **Complejidad accidental** es todo lo demás, la complejidad con la que el
+  equipo de desarrollo no tendría que luchar en el mundo ideal (e.g. complejidad
+  derivada de los problemas de rendimiento y lenguajes suboptimos e
```

Comment:
Cambia el y por coma. En inglés puedes escribir "a and b and c" pero en castellano usamos la conjunción al final